### PR TITLE
Fixed spec converter

### PR DESCRIPTION
**Pull request by chingor13**

### CODE REVIEW

File: clients/storage/lib/google_api/storage/v1/api/objects.ex [hunk 1]
```diff
@@ -441,9 +441,9 @@ defmodule GoogleApi.Storage.V1.Api.Objects do
 
   - connection (GoogleApi.Storage.V1.Connection): Connection to server
   - bucket (String): Name of the bucket in which to store the new object. Overrides the provided object metadata&#39;s bucket value, if any.
-  - upload_type (String): Upload type. Must be \&quot;multipart\&quot;.
-  - metadata (Object): Object metadata
-  - data (String): The file to upload
+  - upload_type (String): Upload type. Must be \&quot;multipart\&quot;
+  - metadata (Object):
```
Comment (dazuma) [inline clients/storage/lib/google_api/storage/v1/api/objects.ex]: Why did these descriptions disappear?

Comment (chingor13) [reply clients/storage/lib/google_api/storage/v1/api/objects.ex]: This was previously generated from a hand-modified OpenAPI json spec and now is using a generated OpenAPI json spec.

Will fix.

Comment (chingor13) [reply clients/storage/lib/google_api/storage/v1/api/objects.ex]: This should now be fixed.

**Comment by chingor13** [comment]
Some upload APIs are incomplete for now. See #7. We'll take this for now and not publish affected APIs yet.

**Pull request merged**